frontend/src/App.js

```diff
@@ -95,7 +105,8 @@ function App() {
             <ListItem>
               <Altimeter
               telemetry={telemetry}
-              command = {()=> { getCommand("DROP_PADA",[])}} />
+            command = {()=> { getCommand("DROP_PADA",['PADA'],['OPEN'])}}
+              resetCommand = {()=> { getCommand("DROP_PADA",['PADA'],['CLOSE'])}} />
```

Comment:
Wrong format. See #67.
There is a command type (`ACTUATE_GROUP`) and then the arguments (`{ group: 'DROP_PADA', state: 'OPEN' }`)
This gets put together into a payload that looks like:
```
{
    commands:
    [
        {
            command: 'ACTUATE_GROUP',
            args:
            [
                group: 'PADA',
                state: 'OPEN' 
            ]
        }
    ]
}
```